### PR TITLE
[WIP] Draggable: Remove handle class when disabled to allow scrolling

### PR DESCRIPTION
**Pull request by scottgonzalez**
Fixes #14779

I'd like to add a test for this before merging which checks if the `touch-action` property is set. I don't have time to do the necessary testing against IE right now though. We also need to handle sortable and selectable, but I wanted to get this started so I don't forget about it.

### CODE REVIEW

**Comment by scottgonzalez** [comment]
Updated the implementation and added tests. If this approach looks good, I'll update the other widgets.

File: ui/widgets/draggable.js [hunk 1]
```diff
@@ -91,6 +94,15 @@ $.widget( "ui.draggable", $.ui.mouse, {
 		}
 	},
 
+	_setOptionDisabled: function( value ) {
+		// Remove touch-action for disabled draggables
```
Comment (arschmitz) [inline ui/widgets/draggable.js]: blank line before comments

Comment (arschmitz) [inline ui/widgets/draggable.js]: But implementation is good

**Comment by BenasPaulikas** [comment]
Is this related to https://forum.jquery.com/topic/jquery-ui-sortable-disables-scroll-on-android ?

When this will be fixed ?

**Comment by aldug** [comment]
I also have been impacted by this behavior.  Is anyone currently working on finishing this?

**Pull request closed without merging**